clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -778,18 +783,23 @@ public void testGroupIdNull() {
 
     @Test
     public void testGroupIdNotNullAndValid() {
+        // close the default consumer
+        shutDown();
```

Comment:
Isn't this anyways going to happen in `afterAll`?

Reply:
The test spins up another consumer so we should shutdown the @BeforeEach one.